### PR TITLE
feat: Table support stack fixed columns

### LINKED ISSUE #28233: 当滚动到某列，Table 组件的某个字段的固定列才生效
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
某个字段初始化的时候不固定(fixed)，当鼠标滚动到某一列时才固定在左边，离开某一列又需要解除冻结，且横向滚动条不能发生变化

### What does the proposed API look like?
autoFixed

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

resolve #28233

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   🔥 Table support stack fixed columns.       |
| 🇨🇳 Chinese |   🔥 Table 支持堆叠固定列。     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -70,3 +70,4 @@ __image_snapshots__/
 
 .eslintcache
 .node-version
+.node
```
Comment (crazyair) [inline .gitignore:73]: 这个是谁产生的文件

File: components/table/demo/fixed-gapped-columns.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    width: 100,
+    dataIndex: 'name',
+    fixed: 'left',
+  },
+  {
+    title: 'Age',
+    width: 100,
+    dataIndex: 'age',
+  },
+  { title: 'Column 1', dataIndex: 'address', fixed: 'left' },
```
Comment (crazyair) [inline components/table/demo/fixed-gapped-columns.tsx:24]: fixed 前面的必须都设置 width 吗

**Review by kiner-tang — DISMISSED** [review]

File: components/table/index.zh-CN.md [hunk 3]
```diff
@@ -87,6 +87,7 @@ const columns = [
 <code src="./demo/tree-table-ellipsis.tsx" debug>树形数据省略情况测试</code>
 <code src="./demo/fixed-header.tsx">固定表头</code>
 <code src="./demo/fixed-columns.tsx">固定列</code>
+<code src="./demo/fixed-gapped-columns.tsx" version="5.14.0">堆叠固定列</code>
```
[the remark this answers is not in the document]
Comment (zombieJ) [reply components/table/index.zh-CN.md:90]: fixed 和 stack fixed 两个例子够用了，剩下的让开发者自己排列组合就好~

File: components/table/demo/fixed-gapped-columns.tsx (hunk 2 above)
Comment (zombieJ) [reply components/table/demo/fixed-gapped-columns.tsx:24]: 有数据的时候会自动算，不是必须的。

File: .gitignore (hunk 1 above)
Comment (zombieJ) [reply .gitignore:73]: cnpm 可以按项目维度使用 node 版本，这个是安装脚本自动加的。

Comment (crazyair) [reply .gitignore:73]: 跟 packageManager 一样啊

Comment (crazyair) [reply .gitignore:73]: cnpm 自动使用 node 版本？确定是 node？

**Review by MadCcc — APPROVED** [review]

**Review by crazyair — APPROVED** [review]

**Pull request merged**